crates/wash-lib/src/app.rs

```diff
@@ -72,11 +72,18 @@ fn resolve_relative_file_paths_in_yaml(
         serde_yaml::Value::String(s)
             if s.starts_with("file://") && s.chars().nth(7).is_some_and(|v| v != '/') =>
         {
-            // Convert the base dir + relative path into a file based URL
-            let full_path = base_dir.as_ref().join(
-                s.strip_prefix("file://")
-                    .context("failed to strip prefix")?,
-            );
+            // Only check existence for relative paths we're resolving
+            let relative_path = s.strip_prefix("file://")
+                .context("failed to strip prefix")?; 
+            
+            let full_path = base_dir.as_ref().join(relative_path);
+            // Only check existence if it was a relative path (starts with ./ or ../)
+            if relative_path.starts_with('.') {
+                (!full_path.exists()).then(|| ()).context(format!(
+                    "referenced relative file does not exist: {}", 
+                    full_path.display()
+                ))?;
+            }
```

Comment:
Ah note that relative paths are actually *any* path that starts in `file://` -- so if you had `file://path/to/file`, we'd expect the relative path to be `path/to/file` (from the base directory).

I think you can change the relative path check to be something like `s.starts_with("file:///")` instead and be a little bit more explicit that we're checking in the case that it *wasn't* absolute. 

That said, I think this code might be better rewritten as a `match` in the first place, so we can cover the `file://` and `file:///` cases really explicitly.